tools/omero/omero_metadata_upload/.shed.yml

```diff
@@ -0,0 +1,16 @@
+categories:
+- Imaging
+name: omero_metadata_upload
+description: Add metadata to an object stored in an OMERO.server using ezomero
+long_description: Create project, dataset, screen and/or attach metadata to an object stored in an OMERO.server
+owner: ufz
+remote_repository_url: https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/omero/omero_metadata_upload
+homepage_url: https://github.com/TheJacksonLaboratory/ezomero
+auto_tool_repositories:
```

Comment:
`auto_tool_repositories` only makes sense with all tools in one dir. Then we do not need to create separate shed.yaml files for each of them. 

In the end it's just a personal preference. If you like this more .. also fine.

Reply:
No problem, it helps to keep everything more organized if we put all omero tools in one dir. I will take care to put everything in one folder and update the shed.yml (https://github.com/Helmholtz-UFZ/galaxy-tools/pull/36/files#r1699114013)